src/app/app.module.ts

```diff
@@ -25,10 +25,11 @@ import { initializeApplication } from "./core/auth/auth.initializer";
 import { errorHandlerFactory } from "./factories/error-handler.factory";
 
 import { swRegistrationOptionsFactory } from "./factories/sw-registration-options.factory";
-import { NgxHotjarModule } from "ngx-hotjar";
```

Comment:
Just checking these removals of hotjar are intended

Reply:
Just re-read the description comment

Reply:
Yep. These 3rd party modules are not compatible with v16 and haven't been updated for some time. The new approach is to use Google Tag Manager, and then add HotJar and GA4 (and any other tags we want in future) from there. This then avoids the need for adding separate tags into the code.